modules/apps/expando/expando-test/src/testIntegration/java/com/liferay/expando/service/test/ExpandoValueLocalServiceTest.java

```diff
@@ -309,15 +354,45 @@ public void testGetSerializableData() throws Exception {
 		Assert.assertEquals("Hi, Joe", enValues[1]);
 	}
 
+	private ServiceRegistration<ModelListener<ExpandoValue>>
+		_registerExpandoValueModelListener() {
+
+		Bundle bundle = FrameworkUtil.getBundle(
+			ExpandoValueLocalServiceTest.class);
+
+		BundleContext bundleContext = bundle.getBundleContext();
+
+		return bundleContext.registerService(
+			(Class<ModelListener<ExpandoValue>>)(Class<?>)ModelListener.class,
+			new BaseModelListener<ExpandoValue>() {
+
+				@Override
+				public void onAfterUpdate(
+						ExpandoValue originalModel, ExpandoValue model)
+					throws ModelListenerException {
+
+					ExpandoRowLocalServiceUtil.getExpandoRowsCount();
+				}
+
+			},
+			new HashMapDictionary<>());
+	}
+
 	private static final Log _log = LogFactoryUtil.getLog(
 		ExpandoValueLocalServiceTest.class);
 
 	private long _classNameId;
 	private Locale _enLocale;
 
+	@Inject
+	private ExpandoRowPersistence _expandoRowPersistence;
```

Comment:
@marianoalvarosaiz These fields are injected but not used in your test code. Are they needed?

Reply:
@dantewang You're right, I forgot to remove it from the previous test.

Reply:
I can resend a new pull without the field once the review is completed.

Reply:
@marianoalvarosaiz Thanks